test/sql.test

```diff
@@ -41,17 +41,27 @@ namespace eval ::qcode::test {
     test sql_sort-1.0 {sql_sort} -setup {
     } -body {
         sql_sort name email user_code 
-    } -cleanup {} -result {name,email,user_code}
+    } -cleanup {} -result {"name","email","user_code"}
```

Comment:
I'm pretty sure that would have been an error under the existing code, this code shouldn't affect that.

The correct usage would be `qc::sql_sort -nulls last -- column_a DESC`, which should continue to work just fine (see test case 1.1)